wremnants/CardTool.py

```diff
@@ -1050,9 +1047,11 @@ def writeLnNSystematics(self):
         # exit this function when a syst is applied to no process (can happen when some are excluded)
         for name,info in self.lnNSystematics.items():
             if self.isExcludedNuisance(name): continue
+            # lowPU does not include PhotonInduced as a process. skip it:
+            if ("lowpu" in self.datagroups.mode) and (name == 'CMS_PhotonInduced'): continue
```

Comment:
In my opinion this should go into setupCombine instead, which means skipping this line for low PU: https://github.com/WMass/WRemnants/blob/main/scripts/combine/setupCombine.py#L748

Reply:
Yeah, this makes more sense, I'll update this -- thanks.